tdrs-backend/tdpservice/parsers/test/conftest.py

```diff
@@ -0,0 +1,434 @@
+"""Package level fixtures."""
+import pytest
+from tdpservice.parsers.test.factories import DataFileSummaryFactory, ParsingFileFactory
+from tdpservice.parsers import util
+
+@pytest.fixture
+def test_datafile(stt_user, stt):
+    """Fixture for small_correct_file."""
+    return util.create_test_datafile('small_correct_file.txt', stt_user, stt)
+
+@pytest.fixture
+def test_header_datafile(stt_user, stt):
```

Comment:
nit: remove test from fixture name, see naming convention [here](https://docs.pytest.org/en/6.2.x/fixture.html)

Reply:
same comment applies to al other fixtures